gui/healthbar.py

```diff
@@ -0,0 +1,19 @@
+import pygame
+
+class HealthBar():
+    def __init__(self, x, y, hp, max_hp):
+        self.x = x
+        self.y = y
+        self.hp = hp
+        self.max_hp = max_hp
+
+
+        def draw(self, hp):
+            self.hp = hp
+            ratio = self.hp / self.max_hp
+            pygame.draw.rect(screen, red, (self.x, self.y, 150, 20))
+            pygame.draw.rect(screen, green, (self.x, self.y, 150 * ratio, 20))
+
+
+player_health_bar = HealthBar(100, screen_height - bottom_panel + 40, player.hp, player.max_hp)
+enemy_health_bar = HealthBar(550, screen_height - bottom_panel + 40, enemy.hp, enemy.max_hp)
```

Comment:
There is no need to make istance of class healthbar beacuse it's only a tool, that will be used inside other modules.